popSimManu.tex

```diff
@@ -1152,10 +1173,10 @@ \section*{Appendix: Calculating coalescence rates}
 should be easier to infer from genomic data than aspects of population demography
 that do not affect effective population size.
 An analogous observation holds for populations of changing size,
-if we define the "coalescence rate" of a given demographic model
+if we define the ``coalescence rate" of a given demographic model
```

Comment:
Shouldn't this be ` ``coalescence rate''`, with two single quotes on the right?

Reply:
This worjs also.